packages/paste-core/components/button/__tests__/button.test.tsx

```diff
@@ -150,132 +149,169 @@ describe('Button', () => {
       expect(() =>
-        shallow(
+        render(
           <Button as="a" href="#" variant="primary" disabled>
             Go to Paste
           </Button>
         )
       ).toThrow();
 
       expect(() =>
-        shallow(
+        render(
           <Button as="a" href="#" variant="primary" loading>
             Go to Paste
           </Button>
         )
       ).toThrow();
+      spy.mockRestore();
     });
 
     it('Throws an error when size=reset is not applied to variant=reset', () => {
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
+      spy.mockImplementation(() => {});
       expect(() =>
-        shallow(
+        render(
           <Button variant="reset" size="small">
             Submit
           </Button>
         )
       ).toThrow();
+      spy.mockRestore();
     });
 
     it('Throws an error when using fullWidth with an icon sizing', () => {
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
+      spy.mockImplementation(() => {});
       expect(() =>
-        shallow(
+        render(
           <Button variant="primary" fullWidth size="icon">
             X
           </Button>
         )
       ).toThrow();
+      spy.mockRestore();
     });
 
     it('Throws an error when using fullWidth with an icon_small sizing', () => {
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
+      spy.mockImplementation(() => {});
       expect(() =>
-        shallow(
+        render(
           <Button variant="primary" fullWidth size="icon_small">
             X
           </Button>
         )
       ).toThrow();
+      spy.mockRestore();
     });
 
     it('Throws an error when not passing children', () => {
-      // @ts-expect-error
-      expect(() => shallow(<Button variant="primary" />)).toThrow();
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
+      spy.mockImplementation(() => {});
+      // @ts-expect-error expect button to throw
+      expect(() => render(<Button variant="primary" />)).toThrow();
+      spy.mockRestore();
     });
 
     it('Throws an error when passing an invalid tabIndex', () => {
-      const originalError = console.error;
-      console.error = jest.fn();
-      // @ts-expect-error
-      expect(() => shallow(<Button variant="primary" tabIndex="-2" />)).toThrow();
-      expect(console.error).toHaveBeenCalled();
-      console.error = originalError;
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
+      spy.mockImplementation(() => {});
+      // @ts-expect-error invalid tabindex
+      expect(() => render(<Button variant="primary" tabIndex="-2" />)).toThrow();
+      expect(spy).toHaveBeenCalled();
+      spy.mockRestore();
```

Comment:
Goodbye enzyme and extremely noisy terminal when throwing expected errors. 

Part of the problem we had is that we silenced jest because it was so noisy we could't figure out what was a real error. To be a little more intentional, and enable us to stop silencing jest, we can mock the console errors, meaning we can actually see when jest is telling us something useful.